src/com/company/Database.java

```diff
@@ -0,0 +1,30 @@
+package com.company;
+
+import java.sql.*;
+
+import static com.company.Main.statement;
+
+public class Database {
+    static String query = "insert into audit values(?,?,now(),?,?,?)";
```

Comment:
It is a good practice to have the SQL statement in capital letters, easier to differentiate between table and column names and statements

```suggestion
    static String query = "INSERT INTO audit VALUES(?,?,now(),?,?,?)";
```

Reply:
changed the query into uppercase.